### ISSUE TITLE
wan2.2 5b training code

### ISSUE THREAD

**Issue by lin076** [aigc-apps/VideoX-Fun#366]
您好，请问是还没有开源wan2.2_Fun_5B_InP 首尾帧训练代码吗？

**Comment by bubbliiiing** [comment]
有开源的呀

**Comment by lin076** [comment]
> 有开源的呀

可以帮忙看一下这个参数配置是正确的吗🌹

CUDA_VISIBLE_DEVICES=0,1 accelerate launch \
  --main_process_port=29502 \
  --num_processes 2 \
  --config_file "config/wan2.2/accelerate.yaml" \
  scripts/wan2.2_fun/train_lora_5b.py \
  --config_path="config/wan2.2/wan_civitai_5b.yaml" \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --train_data_dir=$DATASET_NAME \
  --train_data_meta=$DATASET_META_NAME \
  --video_sample_stride=1 \
  --video_sample_n_frames=81 \
  --train_batch_size=1 \
  --video_repeat=5 \
  --gradient_accumulation_steps=1 \
  --dataloader_num_workers=8 \
  --num_train_epochs=10 \
  --checkpointing_steps=1 \
  --validation_epochs=1 \
  --validation_steps=200 \
  --learning_rate=1e-05 \
  --seed=42 \
  --output_dir="output_dir" \
  --gradient_checkpointing \
  --mixed_precision="bf16" \
  --adam_weight_decay=3e-2 \
  --adam_epsilon=1e-10 \
  --vae_mini_batch=1 \
  --max_grad_norm=1.0 \
  --uniform_sampling \
  --train_mode="inpaint" \
  --boundary_type="full" \
  --lora_skip_name="ffn" \
  --rank=128 \
  --network_alpha=64 \

**Comment by bubbliiiing** [comment]
看起来没什么问题，有报错吗？

**Comment by cat-sun** [comment]
请问这个文件scripts/wan2.2_fun/train_lora_5b.py没看到在哪里啊

**Comment by bubbliiiing** [comment]
scripts/wan2.2_fun/train_lora.py就行
